### ISSUE TITLE
Окно контекстного меню

### ISSUE THREAD

**Issue by alekskdr** [SBProX/Test-Flight#55]
Если вызвать меню правой кнопкой мыши внизу экрана, то периодически часть окна скрыта Панелью инструментов. Монитор 4K, Windows 10, 1920x1080px с масштабированием в 200%
![2021-11-05_11-27-50](https://user-images.githubusercontent.com/574014/140481783-6f224fef-0a63-49a0-858c-0c6f7d0417dd.jpg)

**Comment by SBProX** [comment]
Что-то не очень понятно, какой панелью инструментов скрыто? 
Вы имеете в виду за границами экрана?

Вопрос: зачем на 4k мониторе ставить разрешение FullHD и еще при этом ставить масштабирование 200%?

**Comment by alekskdr** [comment]
> Что-то не очень понятно, какой панелью инструментов скрыто? Вы имеете в виду за границами экрана?
> 
> Вопрос: зачем на 4k мониторе ставить разрешение FullHD и еще при этом ставить масштабирование 200%?

Да, часть меню за границей экрана. К примеру нельзя нажать на пункт "Выйти из SBPro" и приходится щёлкать мышью повторно выше. Но в нормальном случае, все пункты меню отображаются, даже если кликнуть мышью в самом низу окна.

По поводу монитора. Это сделано для того, чтобы на 24" нормально отображалась картинка FullHD. Я такой монитор брал из-за плотности пикселей, чтобы шрифты сглаживались как на macbooke.
